doc/getting_started/setup_fpga.md

```diff
@@ -100,13 +87,13 @@ Now the Vivado GUI opens and loads the project.
 #### Develop with the Vivado GUI
 
 Sometimes it is helpful to use the Vivado GUI to debug a design.
-fusesoc makes that easy, with one small caveat: by default fusesoc copies all source files into a staging directory before the synthesis process starts.
+FuseSoC (the tool Bazel invokes) makes that easy, with one small caveat: by default FuseSoC copies all source files into a staging directory before the synthesis process starts.
```

Comment:
do you think we should remove this section of instructions altogether?
because especially with the bazel flow, i'm not sure the rom `vmem` still ends up at the right place for the fpga build. 

I think it would be fine to build with the bazel flow, and just let the user know where the checkpoint files can be found so that it can be loaded up.

@a-will what do you think?

Reply:
Oh right, perhaps I should file an issue and leave a TODO, saying this is not yet working with the Bazel flow? I imagine a potential solution being we pipe through the fusesoc `--no-export` arg from the bazel rule that currently invokes fusesoc?

Reply:
As of today, bazel does not build bitstreams, so fusesoc arguments are mostly independent. What _does_ need updating is how to construct the fusesoc call and add the required `--BootRomInitFile` and `--OtpCtrlMemInitFile`. These files are no longer in the default location specified in the core file, and the boot ROM has an extra special case where it doesn't end up in the same directory (some hash is part of the directory name).

Finding the boot ROM might need a query like:
```
bazel cquery //sw/device/lib/testing/test_rom:test_rom_fpga_cw310_scr_vmem --output starlark --starlark:expr="target.files.to_list()[0].path" 2> /dev/null
```

Then you'd need the absolute path, and _that_ would be passed as the argument to `--BootRomInitFile`.

(Alternatively, provide some no-op default file that exists in the repo and eliminate that argument entirely. Then have the user do the splicing flow.)

In any case, I think how to find the boot ROM artifact ought to be in the docs.

Reply:
Well, thats not entirely true, there are Bazel rules (and targets) for building OpenTitan FPGA CW310 bitstreams: see [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/bitstream/vivado/BUILD;drc=8fce9b07d969c484eac2af6e73587935c41ae3e9;l=53). However, under the hood, Bazel just makes all files under `hw/` available in the sandbox and then dispatches `fusesoc` with this rule [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/bitstream/vivado/BUILD;drc=8fce9b07d969c484eac2af6e73587935c41ae3e9;l=30).

So technically we could add an arg to that rule to change how this are built in a manner that facilitates debugging if that makes sense?

Reply:
but you are right, bazel itself isn't doing any bitstream building, tho neither is fusesoc. Vivado would be the tool thats doing the actual bitstream building ;)

Reply:
Ah, I was simply under the wrong impression, probably from outdated info. Whoops!

Reply:
I file and issue on this (#13213) and reference said issue in the docs to clean this up.